packages/site/src/layout/NavMenu.module.css

```diff
@@ -117,6 +117,7 @@
   transition: all var(--timing-base) ease-out;
   margin-left: var(--negativeHoverOffset);
   margin-right: var(--negativeHoverOffset);
+  margin-bottom: var(--space-smaller);
```

Comment:
Disclaimer: I'm not a designer 😆  But using this token is my personal preference.
Chris did say `--space-smallest` could probably do the trick but I still see overlap when the button is hovered so that's why I went with `smaller`. If anyone other than Chris feels icky reviewing this I totally understand and we can leave it till he's back.

![Screenshot 2024-12-20 at 5 33 34 PM](https://github.com/user-attachments/assets/5492709b-b6f0-4e67-86f8-0ab3ff5bf162)